server/src/test/java/io/crate/analyze/AlterTableRenameColumnAnalyzerTest.java

```diff
@@ -0,0 +1,154 @@
+package io.crate.analyze;
```

Comment:
Is checkstyle not complaining about missing header?

Reply:
We no longer get the header auto appended when creating a new file and also check style does not check anymore. Need to fix: https://github.com/crate/crate/blob/6f9d72ac44d36cbfff50e8a0813daff7369f600c/server/src/test/java/io/crate/execution/ddl/tables/RenameColumnTaskTest.java#L1